### DISCUSSION TITLE
Lookup Combobox?

### DISCUSSION THREAD

**Discussion by ghost** [AvaloniaUI/Avalonia#15421] [Q&A] [answered]
Hi all.

I am very new in Avalonia as well as in WPF and MAUI.
I am trying to use the Combo box as a Lookup Combo box in Win Forms.
Certainly there must be a way to get the same functionality the Avalonia way.
Google and YouTube were not very helpful in finding any guidance.

I would appreciate any pointers the community might have.

Thanks in advance.

**Comment by timunie** [comment, marked as answer]
Try AutoCompleteBox in Avalonia. I don't know WinForms, so not sure what the differences are.

**Comment by ghost** [comment]
Thanks, I will look into AutoCompleteBox.

I guess my question applies to other components as well.
It must be possible to load a control with items from one table and bind the selected value to a different table. I can't imagine CRUD operations on data without that kind of functionality.

Anyways, thanks for the help Tim.

________________________________

**Reply by timunie** [reply]
Sure, that is what MVVM helps with. Check out Avalonia.Samples for beginner tutorials
